contracts/adapters/Ragequit.sol

```diff
@@ -119,9 +119,7 @@ contract RagequitContract is IRagequit, DaoConstants {
         // it considers the locked loot to be able to calculate the fair amount to ragequit,
         // but locked loot can not be burned.
         uint256 initialTotalSharesAndLoot =
-            bank.balanceOf(TOTAL, SHARES) +
-                bank.balanceOf(TOTAL, LOOT) +
-                bank.balanceOf(TOTAL, LOCKED_LOOT);
+            bank.balanceOf(TOTAL, SHARES) + bank.balanceOf(TOTAL, LOOT);
```

Comment:
Why we don't need to use the LOCKED_LOOT to estimate the total shares issued?

Reply:
we don't need it anymore. Because the way the "challenge" process works has been changed and therefore we don't have that